frontend/src/components/common/PrintCSVButton.tsx

```diff
@@ -16,8 +16,9 @@ import {
   Alert,
```

Comment:
nit: Update the name of the file so that it's easier for devs working with this in the future to pinpoint where it lives (`ExportCSVButton`)

Reply:
If we follow through on the above, we should probably also update the component name

Reply:
I changed the file name and the component name 👍